src/main/java/frc/team2412/robot/subsystems/ArmLEDSubsystem.java

```diff
@@ -0,0 +1,279 @@
+		color2 = ColorSelector.BLACK;
+		color3 = ColorSelector.RED;
+		state = LEDStateSelector.ALLIANCE_COLOR;
+
+		// http request
+		try {
+			client =
+					HttpClient.newBuilder()
+							.connectTimeout(Duration.ofSeconds(CONNECT_TIMEOUT_DURATION))
+							.build();
+		} catch (Exception e) {
+			System.out.println("Failed to initialize http client");
+			System.out.println(e.getMessage());
+		}
+		try {
+			request = getRequest();
+		} catch (Exception e) {
+			System.out.println("Failed to initialize http request");
+			System.out.println(e.getMessage());
+		}
+
+		// logging
+		color1Chooser.setDefaultOption("RED", ColorSelector.RED);
+		color1Chooser.addOption("ORANGE", ColorSelector.ORANGE);
+		color1Chooser.addOption("YELLOW", ColorSelector.YELLOW);
+		color1Chooser.addOption("GREEN", ColorSelector.GREEN);
+		color1Chooser.addOption("BLUE", ColorSelector.BLUE);
+		color1Chooser.addOption("PURPLE", ColorSelector.PURPLE);
+		color1Chooser.addOption("WHITE", ColorSelector.WHITE);
+		color1Chooser.addOption("BLACK", ColorSelector.BLACK);
+
+		color2Chooser.addOption("RED", ColorSelector.RED);
+		color2Chooser.addOption("ORANGE", ColorSelector.ORANGE);
+		color2Chooser.addOption("YELLOW", ColorSelector.YELLOW);
+		color2Chooser.addOption("GREEN", ColorSelector.GREEN);
+		color2Chooser.addOption("BLUE", ColorSelector.BLUE);
+		color2Chooser.addOption("PURPLE", ColorSelector.PURPLE);
+		color2Chooser.addOption("WHITE", ColorSelector.WHITE);
+		color2Chooser.setDefaultOption("BLACK", ColorSelector.BLACK);
+
+		color3Chooser.setDefaultOption("RED", ColorSelector.RED);
+		color3Chooser.addOption("ORANGE", ColorSelector.ORANGE);
+		color3Chooser.addOption("YELLOW", ColorSelector.YELLOW);
+		color3Chooser.addOption("GREEN", ColorSelector.GREEN);
+		color3Chooser.addOption("BLUE", ColorSelector.BLUE);
+		color3Chooser.addOption("PURPLE", ColorSelector.PURPLE);
+		color3Chooser.addOption("WHITE", ColorSelector.WHITE);
+		color3Chooser.addOption("BLACK", ColorSelector.BLACK);
+
+		stateChooser.addOption(
+				"Disable Arm LEDs",
+				new InstantCommand(
+						() -> {
+							enableLED(false);
+						}));
+		stateChooser.addOption(
+				"Use Alliance Color (RED/BLUE)",
+				new InstantCommand(
+						() -> {
+							enableLED(true);
+							setLEDAlliance();
+						}));
+		stateChooser.addOption(
+				"Use Custom Colors",
+				new InstantCommand(
+						() -> {
+							enableLED(true);
+							setLEDCustom();
+						}));
+		stateChooser.setDefaultOption(
+				"Use Alliance Color (RED/BLUE)",
+				new InstantCommand(
+						() -> {
+							enableLED(true);
+							setLEDAlliance();
+						}));
+
+		armLEDTab = Shuffleboard.getTab("Arm LED");
+
+		armLEDTab.add("Color 1", color1Chooser).withPosition(0, 0).withSize(2, 1);
+		armLEDTab.add("Color 2", color2Chooser).withPosition(2, 0).withSize(2, 1);
+		armLEDTab.add("Color 3", color3Chooser).withPosition(4, 0).withSize(2, 1);
+		armLEDTab.add("State", stateChooser).withPosition(6, 0).withSize(2, 1);
```

Comment:
I don't think we actually use the selected value of `color3Chooser` or `stateChooser`, and I think this was behavior we didn't test yesterday. (We do use `color3Chooser` to set `color3` once but `color3` isn't used either)

Reply:
Yeah, the choosers don't work currently as we saw yesterday. Will need to be addressed in a future pr. But right now we at least do have the alliance colors during teleop and green during auto

Reply:
For the choosers to work we have to monitor NT for a change in value of the selection and then run the instant command that was selected